Mathlib/CategoryTheory/Abelian/Projective/Dimension.lean

```diff
@@ -88,6 +88,13 @@ lemma Limits.IsZero.hasProjectiveDimensionLT_zero (hX : IsZero X) :
 instance : HasProjectiveDimensionLT (0 : C) 0 :=
   (isZero_zero C).hasProjectiveDimensionLT_zero
 
+lemma isZero_of_hasProjectiveDimensionLT_zero (hX : HasProjectiveDimensionLT X 0) : IsZero X := by
+  letI := HasExt.standard C
+  rw [hasProjectiveDimensionLT_iff] at hX
+  exact (IsZero.iff_id_eq_zero X).mpr <|
+    ((Ext.addEquiv₀.symm_apply_eq ).mp (hX 0 (le_refl 0) (Ext.addEquiv₀.symm (𝟙 X)))).trans
+      (Ext.addEquiv₀ (X := X)).map_zero
```

Comment:
I would find this more readable (and the parameter should be `[...]` as this is a typeclass):
```suggestion
lemma isZero_of_hasProjectiveDimensionLT_zero [HasProjectiveDimensionLT X 0] : IsZero X := by
  letI := HasExt.standard C
  rw [IsZero.iff_id_eq_zero]
  apply Ext.homEquiv₀.symm.injective
  simpa only [Ext.homEquiv₀_symm_apply, Ext.mk₀_zero]
    using Abelian.Ext.eq_zero_of_hasProjectiveDimensionLT _ 0 (by rfl)
```